src/overlays/actors/ovl_En_Dinofos/z_en_dinofos.h

```diff
@@ -5,9 +5,43 @@
 
 struct EnDinofos;
 
+typedef void (*EnDinofosActionFunc)(struct EnDinofos*, GlobalContext*);
+
 typedef struct EnDinofos {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x500];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[21];
+    /* 0x206 */ Vec3s morphTable[21];
+    /* 0x284 */ EnDinofosActionFunc actionFunc;
+    /* 0x288 */ u8 unk_288;
+    /* 0x289 */ u8 unk_289;
+    /* 0x28A */ u8 unk_28A;
+    /* 0x28B */ u8 unk_28B;
+    /* 0x28C */ s16 unk_28C;
+    /* 0x28E */ s16 unk_28E;
+    /* 0x290 */ s16 unk_290;
+    /* 0x290 */ s16 unk_292;
+    /* 0x294 */ UNK_TYPE1 unk_294[0x4];
+    /* 0x298 */ s16 unk_298;
+    /* 0x29A */ UNK_TYPE1 unk_29A[0x6];
```

Comment:
```suggestion
    /* 0x294 */ UNK_TYPE1 unk_294[4];
    /* 0x298 */ s16 unk_298;
    /* 0x29A */ UNK_TYPE1 unk_29A[6];
```

Reply:
Why only these 2 as decimal but all the others in hex? Shouldn't that be consistent?

Reply:
My motivation is that hex in single digits don't really need the `0x` in front, but I'll admit even I've been a little inconsistent with that sometimes. No preference either way